virtualmachine/vsphere/util.go

```diff
@@ -802,7 +802,8 @@ var halt = func(vm *VM) error {
 	vmo := object.NewVirtualMachine(vm.client.Client, vmMo.Reference())
 	poweroffTask, err := vmo.PowerOff(vm.ctx)
 	if err != nil {
-		return fmt.Errorf("error creating a poweroff task on the vm: %s", err)
+		return fmt.Errorf(
```

Comment:
is formatting the only change here?
%s->%v

Reply:
80 character limit, i was trying to follow this. Trying to correct this wherever i find exceeding limit.